### PR TITLE
Components: Add Docs and Example to PlansSkipButton

### PR DESCRIPTION
**Pull request by ockham**
Follow-up to #24312.

image

### Testing Instructions

Visit http://calypso.localhost:3000/devdocs/design/plans-skip-button and make sure that nothing breaks.
If you click the 'Start with Free' button, you'll see a console message.

### CODE REVIEW

File: client/components/plans/plans-skip-button/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Plans Skip Button.
+
+Render a button that allows a user to forego (paid) plans selection during creation of a new site, and use the Free plan instead.
+
+## Props 
+
+* **isRtl** — (optional) Whether the button should be rendered for a Right-to-Left locale.
```
Comment (sirreal) [inline client/components/plans/plans-skip-button/README.md]: It's unfortunate that devs need to remember to pass this to get proper behavior. Maybe worth connecting the component, definitely worth investigating a better solution.

File: client/components/plans/plans-skip-button/docs/example.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	handleClick = () => {
+		console.log( 'PlansSkipButton was clicked' );
+	};
+
+	render() {
+		return <PlansSkipButton onClick={ this.handleClick } />;
+	}
+}
+
+export default PlansSkipButtonExample;
```
Comment (sirreal) [inline client/components/plans/plans-skip-button/docs/example.js]: connect `isRtl`?

File: client/components/plans/plans-skip-button/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import PlansSkipButton from 'components/plans/plans-skip-button';
+
+class PlansSkipButtonExample extends PureComponent {
+	static displayName = 'PlansSkipButton';
+
```
Comment (sirreal) [inline client/components/plans/plans-skip-button/README.md]: Let's add `isRtl`

**Comment by ockham** [comment]
Thanks for reviewing! Addressed copy feedback; going to `connect()` to the `isRtl` selector in a follow-up.

**Comment by drw158** [comment]
We now have a [readme template](https://github.com/Automattic/wp-calypso/blob/8e11380288256d4dd4c3dcaae99420f16dacd89b/docs/component-readme-template.md) for UI components. When you get a chance, please update the PlansSkipButton to use the template. We also have a [new component checklist](https://github.com/Automattic/wp-calypso/blob/a9d192be25db265921960de680885046edfc5869/docs/new-component-checklist.md) for any future additions.

**Pull request merged**